rosey/src/runners/builder/html.rs

```diff
@@ -564,6 +564,22 @@ impl<'a> RoseyPage<'a> {
         };
         meta_node.insert_after(NodeRef::new_text(&indentation));
 
+        let mut attributes = BTreeMap::new();
+        attributes.insert(
+            ExpandedName::new("", "rel"),
+            Attribute {
+                prefix: None,
+                value: String::from("alternate"),
+            },
+        );
+        let locale_key_node = NodeRef::new_element(
+            QualName::new(None, ns!(html), local_name!("link")),
+            attributes,
+        );
+        self.link_tags.push(locale_key_node.clone());
+        meta_node.insert_after(locale_key_node.clone());
+        meta_node.insert_after(NodeRef::new_text(&indentation));
+
         for _i in 0..self.translations.len() {
```

Comment:
```suggestion
        for _i in 0..=self.translations.len() {
```

We only need to ensure there are enough of these link tags for each variant to hydrate later, so we just need to add one extra iteration of that loop.

Reply:
@bglw Fixed, thanks for that tips, still learning Rust 😅